### PR TITLE
Style guided refactor

### PR DESCRIPTION
**Pull request by mohammadkhavari**

### CODE REVIEW

File: highlite/src/App.vue [hunk 1]
```diff
@@ -36,55 +30,16 @@ export default {
     CustomizeToolbar,
   },
   data: () => ({}),
```
Comment (javadvhd) [inline highlite/src/App.vue]: delete this

File: highlite/src/App.vue [hunk 2]
```diff
@@ -11,19 +11,13 @@
   </v-app>
 </template>
 
-
 <script>
-// <Drawer />
-import debounce from 'debounce'
 import AppBar from './components/TheAppBar'
```
Comment (javadvhd) [inline highlite/src/App.vue:16]: // components

File: highlite/src/components/TheAppBar.vue [hunk 3]
```diff
@@ -82,14 +82,13 @@ export default {
...
-  width: 40px;
+  min-width: 40px;
   height: 40px;
-  background-color: ;
   border-radius: 25px;
 }
 
-.c--appBar-icon {
-}
+/* .c--appBar-icon {
```
Comment (javadvhd) [inline highlite/src/components/TheAppBar.vue]: del

File: highlite/src/components/TheAppBar.vue [hunk 4]
```diff
@@ -133,8 +132,8 @@ export default {
   .c--appBar-header-input {
     width: 150px;
   }
-  .c--appBar-header-panel {
-  }
+  /* .c--appBar-header-panel {
```
Comment (javadvhd) [inline highlite/src/components/TheAppBar.vue]: del

File: highlite/src/components/TheEditor.vue [hunk 5]
```diff
@@ -0,0 +1,127 @@
+<template>
+  <div class="editor-container">
```
Comment (javadvhd) [inline highlite/src/components/TheEditor.vue]: c--

File: highlite/src/components/TheEditor.vue [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+      <ToolBar :editor="this.editor"/>
+    </EditorMenuBar>
+    <ToggleableBar :commands="this.editor.commands"/>
+    <v-divider></v-divider>
+    <EditorContent class="editor-panel" :editor="editor" />
+  </div>
+</template>
+
+<script>
+import { Editor, EditorContent, EditorMenuBar } from 'tiptap'
```
Comment (javadvhd) [inline highlite/src/components/TheEditor.vue:14]: //components

File: highlite/src/components/TheEditorToggleableBar.vue [hunk 7]
```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="toggleable-toolbar">
+    <Colors v-if="toggleablePanel == 'color-panel'" :commands="commands" />
+    <Heading v-if="toggleablePanel == 'heading-panel'" :commands="commands" />
+    <Link v-if="toggleablePanel == 'link-panel'" :commands="this.commands" />
```
Comment (javadvhd) [inline highlite/src/components/TheEditorToggleableBar.vue]: this ?

File: highlite/src/components/TheEditorToggleableBarHeading.vue [hunk 8]
```diff
@@ -0,0 +1,32 @@
+<template >
+  <div class="heading-container">
+    <BaseButton @click="this.toggleDrawer" imageSrc="menu.svg" />
+    <BaseButton stretched @click="commands.heading({ level: 2 })">ADD</BaseButton>
+    <BaseButton stretched>PREV</BaseButton>
+    <BaseButton stretched>NEXT</BaseButton>
+  </div>
+</template>
+<script>
+import { mapState, mapMutations } from 'vuex'
```
Comment (javadvhd) [inline highlite/src/components/TheEditorToggleableBarHeading.vue]: remove mapstate & data

File: highlite/src/components/TheEditorToggleableBarLink.vue [hunk 9]
```diff
@@ -0,0 +1,52 @@
+<template>
+  <div>
+    <div class="link-container">
```
Comment (javadvhd) [inline highlite/src/components/TheEditorToggleableBarLink.vue]: c--

File: highlite/src/components/TheEditorToggleableBarLink.vue [hunk 10]
```diff
@@ -0,0 +1,52 @@
+<template>
+  <div>
```
Comment (javadvhd) [inline highlite/src/components/TheEditorToggleableBarLink.vue]: remove this

File: highlite/src/helpers/editorToolsHandler.js [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+    name: 'redo',
+    active: () => false,
+    command: editor.commands.redo,
+    imageSrc: 'redo.svg',
+  },
+]
+
+export default (editor, togglePanelTo, customizeArray) => {
+  return getAllTools(editor, togglePanelTo).filter((tool, index)=>{
+    return customizeArray[index].able
```
Comment (javadvhd) [inline highlite/src/helpers/editorToolsHandler.js]: arrow

File: highlite/src/store.js [hunk 12]
```diff
@@ -1,15 +1,20 @@
...
-
+import { eventBus } from './components/bus'
+import { setInitialData } from './helpers/typesUtils'
+import { save, fetch, update } from './helpers/requestHandler'
+import autoSavePlugin from './helpers/Plugins/autoSavePlugin'
+import fetchInitialDataPlugin from './helpers/Plugins/fetchInitialDataPlugin'
 Vue.use(Vuex)
 
 export default new Vuex.Store({
+  plugins: [autoSavePlugin, fetchInitialDataPlugin],
```
Comment (javadvhd) [inline highlite/src/store.js]: put at the end

**Review by javadvhd — CHANGES_REQUESTED** [review]

**Review by mohammadkhavari — COMMENTED** [review]
resolved

**Pull request merged**